src/main/java/com/google/sps/data/CommentTable.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+public class CommentTable {
+
+    private ArrayList<Comment> commentTable;
+
+    public CommentTable() {
+        commentTable = new ArrayList<Comment>();
+    }
+
+    public void addEntry(String name, String comment) {
+        if (name == null || String.isEmpty(name))
+            name = "Anonymous";
+
+        if (comment == null || String.isEmpty(comment))
+            return;
+
+        commentTable.add(new Comment(name, comment));
+    }
+
+    private class Comment {
+        String name;
+        String comment;
+
+        public Comment(String _name, String _comment) {
```

Comment:
nit: From Google's Style Guide "5.1 Rules common to all identifiers" - https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names

The standard for this case at Google is to name your parameter name and use `this.name = name`.